01.md

```diff
@@ -43,6 +43,14 @@ To obtain the `event.id`, we `sha256` the serialized event. The serialization is
 ]
 ```
 
+To prevent implementation differences from creating a different event ID for the same event, the following rules MUST be followed while serializing:
+- No whitespace, line breaks or other unnecessary formatting should be included in the output JSON.
+- No characters except the following should be escaped, and instead should be included verbatim:
+  - The line break, `0x0A`, as `\n`
+  - A double quote, `0x22`, as `\"`
+  - A backspace, `0x5C`, as `\\`
+- UTF-8 should be used for encoding.
+
```

Comment:
I think this is missing other required escape characters in JSON - backspace, form feeds, carriage returns, and tabs.